api/v1/cluster_types.go

```diff
@@ -1162,10 +1162,24 @@ const (
 	SynchronousReplicaConfigurationMethodAny = SynchronousReplicaConfigurationMethod("any")
 )
 
+// DataDurabilityMethod specifies how strictly to enforce synchronous replication
+// when cluster instances are unavailable. Options are `required` or `preferred`.
+type DataDurabilityMethod string
+
+const (
+	// DataDurabilityMethodRequired means that data durability is strictly enforced
+	DataDurabilityMethodRequired DataDurabilityMethod = DataDurabilityMethod("required")
```

Comment:
I don't think we need to cast the constants ... this line looks like Java (ha)
```suggestion
	DataDurabilityMethodRequired DataDurabilityMethod = "required"
```